fe/fe-core/src/main/java/com/starrocks/alter/MaterializedViewManager.java

```diff
@@ -0,0 +1,94 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.alter;
+
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import com.starrocks.catalog.MaterializedView;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+// a manager for materialized view
+// it has the following functions:
+// 1. manage the relations between manterialized view and base tables
+public class MaterializedViewManager {
+    // mv name -> materialized view
+    private Map<String, MaterializedView> materializedViews;
+
+    // table id -> mv set
+    // every table that has mvs will be recorded here
+    private Map<Long, Set<MaterializedView>> materializedViewsForTable;
```

Comment:
how about directly push the MV ids to OlapTable?   then we needn't this map.

Reply:
I pretend to add the relationship here. I do not want to add fields to OlapTable for back-compatible reasons.

Reply:
I will move the relations to OlapTable and delete MaterializedViewManager